javascript/net/grpc/web/grpcwebclientbase.js

```diff
@@ -227,10 +227,12 @@ class GrpcWebClientBase {
    * @param {boolean} useUnaryResponse
    */
   static setCallback_(stream, callback, useUnaryResponse) {
+    var responseReceivedSet = false;
```

Comment:
Thanks for the contributions!

Just a nit: can we rename this variable to `isResponseReceived`?

Reply:
yeah sure